requirements.txt

```diff
@@ -20,6 +20,7 @@ imagesize==1.2.0
 importlib-metadata==1.6.1
 jeepney==0.4.3
 Jinja2==2.11.2
+jsonschema==3.2.0
```

Comment:
It's not enough to add the dependency on `jsonschema` here because `requirements.txt` is intended for developers. You have to add it to the `install_requires` param in `setup.py` too so one can install `qrest` from PyPI.

Reply:
Done